barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.cpp

```diff
@@ -2654,17 +2641,8 @@ template <typename Arithmetization> void UltraCircuitBuilder_<Arithmetization>::
     // add the index/timestamp values of the last sorted record in an empty add gate.
     // (the previous gate will access the wires on this gate and requires them to be those of the last record)
     const auto& last = sorted_ram_records[ram_array.records.size() - 1];
-    create_big_add_gate({
-        last.index_witness,
-        last.timestamp_witness,
-        this->zero_idx,
-        this->zero_idx,
-        0,
-        0,
-        0,
-        0,
-        0,
-    });
+    create_dummy_gate(blocks.main, last.index_witness, last.timestamp_witness, this->zero_idx, this->zero_idx);
```

Comment:
This was formerly an add gate but all of the selectors were set to zero so the constraint was trivially satisfied.